README.md

```diff
@@ -44,6 +44,11 @@ parameter | explanation
 `stealth.user_agent` |  A string representing a user agent that is used by the stealth module as a user agent value. It can be an empty string. If this query parameter does not exist, it is treated as not enabled.
 `stealth.ip` |  A string representing a IP address that is used by the stealth module as a ip address. If this query parameter does not exist, it is treated as not enabled.
 `stealth.dpi` | Can be `true` or `false`.
+`stealth.block_trackers` | Can be `true` or `false`.
```

Comment:
this stealth option enables AdGuard Tracking Protection filter so there is no need to pass it to the ReportsWebApp

Reply:
![Screenshot 2023-10-20 at 10 18 42](https://github.com/AdguardTeam/ReportsWebApp/assets/2804667/2182c324-7af3-4bb7-871d-7d6b64277fb5)

According to the new design, we display this setting in step 4

Reply:
maybe better to check if filter id `3` is passed? 
the same for "Remove tracking parameters from URLs" (related filter id is 17)